CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+### citus v8.1.0 (December 17, 2018) ###
+
+* Turns on ssl by default for new installations of citus
+
+* Restricts SSL Ciphers to TLS1.2 and above
+
+* Adds support for INSERT INTO SELECT..ON CONFLICT/RETURNING via coordinator
+
+* Adds support for round-robin task_assignment_policy for queries
```

Comment:
This item should include `reference tables`, because this is already a Citus feature, we're only expanding it for reference tables. If we don't have it, it'd be confusing.

Maybe: `Support round-robin task assignment for queries to reference tables.`

Reply:
I needed to shorten the description, and your _shortened_ version makes much more sense here